src/components/ProposalListTable.tsx

```diff
@@ -42,16 +42,38 @@ const ProposalListTableRow = ({
 };
 
 interface ProposalListTableProps {
-	columns: string[];
+	columns?: string[];
 	fieldNames: Record<string, string>;
 	proposals: FrontEndProposal[];
 	wrap?: boolean;
 }
 
+// For now, we are hard-coding this list.
+// In the future, this will be user-configurable.
```

Comment:
You can delete this comment. I know it was written by me / from the prior context, but I shouldn't have included it here and we should remove.